.github/workflows/pr-build.yml

```
@@ -264,35 +265,68 @@ jobs:
+          context: .
+          file: Dockerfile.kernel
+          push: true
+          tags: ${{ steps.meta-kernel.outputs.tags }}
+          labels: ${{ steps.meta-kernel.outputs.labels }}
+          provenance: false
+          sbom: false
+
+      - name: Get image sizes
         if: needs.prepare.outputs.build_needed == 'true'
         id: image-size
         run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:4:28: Double quote to prevent globbing and word splitting

(shellcheck)